journalbeat/magefile.go

```diff
@@ -80,11 +191,26 @@ func Package() {
 	defer func() { fmt.Println("package ran for", time.Since(start)) }()
 
 	mage.UseElasticBeatPackaging()
+	customizePackaging()
+
 	mg.Deps(Update)
 	mg.Deps(CrossBuild, CrossBuildXPack, CrossBuildGoDaemon)
 	mg.SerialDeps(mage.Package, TestPackages)
 }
 
+// customizePackaging adds dependencies of journalbeat.
+func customizePackaging() {
+	for _, args := range mage.Packages {
+		switch args.OS {
+		case "linux":
+			if args.Types[0] == mage.Deb {
+				args.Spec.AddDebDependency(currentLibsystemdDev)
```

Comment:
I wouldn't expect most systems to have `-dev` packages installed. I think you want a dependency on `systemd`.

Reply:
sorry, i did not want to push this code yet